costar_dataset/block_stacking_reader_torch.py

```diff
@@ -408,6 +417,49 @@ def preprocess_numpy_input(x):
     return x
 
 
+def preprocess_numpy_input_torch(x):
```

Comment:
The name of this original function in keras is misleading, and I suspect the confusing name misled you. `preprocess_numpy_input_torch` only has this name because some paper originally implemented in torch (original lua torch, rather than pytorch), happened to use this preprocessing method. 

What was the reason you added this?